### DISCUSSION TITLE
Learned a new skill over Christmas break

### DISCUSSION THREAD

**Discussion by bcopeland** [cardonabits/haxo-hw#63] [Show and tell]
My haxophone arrived a couple of weeks ago.  I've never played a woodwind let alone a sax, but set a goal of learning the Hank Mobley head to "Remember" (which has just 7 notes if you don't count the octaves).  Happy to report that just a couple of hours' practice spread out over that two weeks was enough to get a reasonable version:

https://youtu.be/gcF8FIdH2a8

Looking forward to playing with the software side now

**Comment by jcard0na** [comment]
Very nice, @bcopeland !
It's a great tune... and a great interpretation!

Thanks for sharing!
